src/components/profile-view/profile-view.jsx

```diff
@@ -1,24 +1,31 @@
 import React, { useState, useEffect } from 'react';
-import { Card, CardGroup, Form, Button, ButtonGroup, ButtonToolbar, Row, Col, Image } from 'react-bootstrap';
+import { Card, CardGroup, Form, Button, ButtonGroup, ButtonToolbar, Row, Col, Image, Figure } from 'react-bootstrap';
 import PropTypes from 'prop-types';
 import './profile-view.scss';
 import { BrowserRouter as Router, Route, Redirect, Link } from "react-router-dom";
 import axios from 'axios';
+import FigureImage from 'react-bootstrap/esm/FigureImage';
+
 
 export function ProfileView(props) {
     console.log('props: ', props);
 
-    // ---------  VARIABLES FOR UPDATE USER FORM
+
     const [username, setUsername] = useState(props.userData.Username);
     const [password, setPassword] = useState('');
     const [email, setEmail] = useState(props.userData.Email);
     const [birthday, setBirthday] = useState(props.userData.Birthday);
-
-
+    const token = localStorage.getItem('token');
+    const userInfo = {
+        Username: username,
+        Password: password,
+        Email: email,
+        Birthday: birthday
+    }
     const currentUser = localStorage.getItem('user');
     const { movies, userData } = props;
     console.log(userData, 'userdata');
-    const favorites = movies.filter(movie => userData.FavouriteMovies.indexOf(movie._id) > -1);
+    const favourites = movies.filter(movie => userData.FavouriteMovies.indexOf(movie._id) > -1);
```

Comment:
Small bug here. if a user has not added any FavoriteMovies, then `userData.FavouriteMovies` will be undefined and then `userData.FavouriteMovies.indexOf` will throw an error. try
```js
const favourites = movies.filter(movie => (
  userData.FavouriteMovies && userData.FavouriteMovies.indexOf(movie._id) > -1
));
```

Reply:
I tested the app by : removing all favourite movies from a user / logging out / logging back in / going to the profile-view where no favourite movies can be seen and it doesn't throw error. 
I implemented your && operator suggestion and can see how this can be a problem otherwise, thanks for thorough check.